tools/image-builder/build_bond.zsh

```diff
@@ -0,0 +1,82 @@
+#!/bin/zsh
+
+set -eux
+setopt shwordsplit
+
+HOME=$1
+FLAVOR=$2
+BOOST=${3:-}
+
+BUILD_PATH=/root/build/$FLAVOR
```

Comment:
What's the purpose of having separate build dirs for each flavor? One of the goals of containerizing this, at least for me, is creating a stateless build - every time you run the container with one flavor, `/root/build` shouldn't exist.

Reply:
Switched to a `/root/build` for all flavors.